flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/runtime/utils/StreamingWithAggTestBase.scala

```diff
@@ -63,9 +63,11 @@ object StreamingWithAggTestBase {
 
   @Parameterized.Parameters(name = "LocalGlobal={0}, {1}, StateBackend={2}")
   def parameters(): util.Collection[Array[java.lang.Object]] = {
+    // To avoid too long testing time, we not test all HEAP_BACKEND.
+    // If HEAP_BACKEND has changed, please open it manually for testing.
```

Comment:
Comments like this get lost in the code base. Tests like this should be moved to a special manual tests package.